src/host_toolchains.py

```diff
@@ -26,8 +26,8 @@
 SETUP_TOOLCHAIN = os.path.join(CR_BUILD_DIR, 'toolchain', 'win',
                                'setup_toolchain.py')
 V8_SRC_DIR = os.path.join(WORK_DIR, 'v8', 'v8')
```

Comment:
> Did the v8 version have some useful difference?

As far as I know, there is no useful difference, just out-dated.

I believe this out-dated vs_toolchain.py is pulling the older version of VS 2015, so when `ninja` spawns 50 jobs to build LLVM, 50 of them fail with compile error (see [here](https://wasm-stat.us/builders/windows/builds/8716/steps/LLVM/logs/stdio/text))